src/components/moviesReviews.js

```diff
@@ -0,0 +1,74 @@
+import { Component } from 'react';
+import axios from 'axios';
+
+import './movieReviews.css';
+
+class MoviesReviews extends Component { 
+
+  _isMounted = false;
+    constructor(props) {
+        super(props);
+     
+        this.state = {
+          movies: [],
+        };
+      }
+      
+      componentDidMount(){
+        this._isMounted = true;
+        const moviesId = this.props.match.params.movieId;
+        console.log("moviesId " + moviesId);
+        axios.get(`
+        https://api.themoviedb.org/3/movie/${moviesId}/reviews?api_key=ed4f70d18fa8cbc03fb59ce04847e1bc`)
```

Comment:
same API key is being used in multiple components, store it in a different file as constant and import where required